src/cascadia/TerminalConnection/ConhostConnection.cpp

```diff
@@ -4,7 +4,11 @@
 #include "pch.h"
 #include "ConhostConnection.h"
 #include "windows.h"
+#include "wil/common.h"

```

Comment:
I think we already have wil/common, string_view, algorithm and type_traits in our precompiled header through `pch.h` -> `LibraryIncludes.h`!

Reply:
I'm used to including the related headers. It's difficult for me to find the way across other header files to figure out which have been already included over detours.
Before I push the next commit I should probably check if it compiles without string_view, algorithm, and type_traits included directly.